projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-grid.component.html

```diff
@@ -190,6 +191,10 @@
     <div class="igx-grid__scroll-end" [style.float]='"right"' [style.width.px]='pinnedWidth' [style.min-width.px]='pinnedWidth' [hidden]="pinnedWidth === 0 || isPinningToStart"></div>
 </div>
 
+<div class="igx-grid__addrow-snackbar">
+    <igx-snackbar #addRowSnackbar [actionText]="snackbarActionText" displayTime='2000'>{{snackbarLabel}}</igx-snackbar>
```

Comment:
might have missed that in the other prs - let's at least have this 2000 as a const in the grid.base instead of a number in each template